src/_routes/index.js

```diff
@@ -4,7 +4,9 @@ import TickTradeRoutes from './TickTradeRoutes';
 import DesktopRoutes from './DesktopRoutes';
 import CommonRoutes from './CommonRoutes';
 
-const childRoutes = [].concat(DesktopRoutes).concat(TickTradeRoutes).concat(CommonRoutes);
+import HelloWorldRoute from './HelloWorldRoute'
+
+const childRoutes = [].concat(DesktopRoutes).concat(TickTradeRoutes).concat(CommonRoutes).concat(HelloWorldRoute);
```

Comment:
You can't concat an object to an array, you would have to do .concat([HelloWorldRoute])